### ISSUE TITLE
Glitched reply section

### ISSUE THREAD

**Issue by Deathstalkr1** [VueTubeApp/VueTube#467]
### Steps to reproduce

Open any video in vuetube.
Open comments
Open any comment and view replies.

### Expected behavior

Show replies from different people

### Actual behavior

Shows the top comment until reply chain ends

### VueTube version

51b0be2

### Android version

Android 12

### Device

Samsung Galaxy S21 Ultra 5G

### Other details

![Screenshot_20220728-211524_VueTube](https://user-images.githubusercontent.com/100892324/181663034-4d9b637c-f830-4ad2-9c65-065498472d05.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to unstable version **[Latest](https://vuetube.app/install/)**.
- [X] I will fill out all of the requested information in this form.

**Comment by 404-Program-not-found** [comment]
Yup, this is known. Replies have not been properly implemented and will not be until the release of the extractor

**Comment by gayolGate** [comment]
There are too many comments why you had to select THAT one

**Comment by Frontesque** [comment]
duplicate of #462

**Comment by PickleNik** [comment]
> duplicate of #462

I don't think that's a dup, it just refers to replies being unfinished
